scripts/operations/configuration/backup_cfs_config_comp.sh

```diff
@@ -55,12 +55,15 @@ usage()
    echo "All parameters are optional and the values will be determined automatically if not set."
    echo
    echo "Usage: backup_cfs_config_comp.sh [ -b base_directory | -d target_directory ]"
+   echo "                                 [--comp-only | --cnfg-only]"
    echo
    echo "By default, a backup directory will be created as a subdirectory of /root."
    echo
    echo "Options:"
    echo "-b base_directory    Create the backup directory under base_directory instead of /root"
    echo "-d target_directory  Create the backup files in target_directory instead of creating a new directory."
+   echo "--comp-only           Only back up components"
+   echo "--cnfg-only           Only back up configurations"
```

Comment:
The only reason was so that it would be the same length of the other argument, which made the argument parsing easier later (I could rely on the comp/cnfg part of the variable being in the same indices). I agree it would definitely not be worth doing just for the length savings. I originally had them as "cmp" and "cfg", but that felt too short in the case of "cmp". 

Of course, it's not really a big deal if they're different lengths. It just seemed like something that is unlikely to be used very much anyway, so I just took the path of least resistance. But it is kind of ugly, I will admit.

Reply:
I'm good with leaving it. I suspect this script isn't going to get much usage outside of copy-pasting its invocations from the docs anyway.